Fix sed arguments to be compatible with BSD sed as well.
This bug needs to be reported to the developers but I am unable to locate
their bug tracker.
--- Makefile.orig	2019-09-21 19:07:07.000000000 -0700
+++ Makefile	2019-09-21 19:07:33.000000000 -0700
@@ -4,7 +4,7 @@
      sig2dot springgraph gpgwrap gpgdir keyart gpg-key2latex
 
 all:
-	sed -i "s/@@VERSION@@/$(DEB_VERSION_UPSTREAM)/g" \
+	sed -i~ "s/@@VERSION@@/$(DEB_VERSION_UPSTREAM)/g" \
             caff/caff \
             caff/pgp-clean \
             caff/pgp-fixkey \
